### PR TITLE
Issue 4993

### PR DESCRIPTION
**Pull request by julienanquetil**
**Description (for Contributor and Core Developer)**
Allow numeric value as code in Simple Select attribute

Issue #4993 

Thanks @solivier

### CODE REVIEW

**Comment by solivier** [comment]
hi @julienanquetil,

Thanks a lot for that PR :tada: . Are you interested to add a behat to cover this case ? I can help you to do it if you want, I could also do it for you if you are not comfortable to add a behat :)

Have a nice day.
Olivier.

**Comment by julienanquetil** [comment]
Hi @solivier 
I can try but never done it before... so if you can "show me the right way" i can try to do something ;)
Thanks

**Comment by solivier** [comment]
Hi @julienanquetil,

Yes sure :) you could try to follow the doc here https://docs.akeneo.com/1.5/reference/best_practices/core/behat.html#install-behat it explains how to create the "behat environment" and then our behat for the imports are here https://github.com/akeneo/pim-community-dev/blob/master/features/import/product/import_products.feature

You could for example add a new behat with the manufacturer attribute witch is a simpleselect attribute.
And write something like this in your behat

`
Scenario: Successfully import a csv file for a product with a numeric data for an simple select attribute
Given the "clothing" catalog configuration
    And the following CSV file to import:
      """
      sku;manufacturer
      renault-kangoo;70
      """
And the following job "csv_clothing_product_import" configuration:
      | filePath | %file to import% |
    When I am on the "csv_clothing_product_import" import job page
    And I launch the import job
    And I wait for the "csv_clothing_product_import" job to finish
    Then there should be 1 products
`
 And also check if your product has the correct value imported for the simple select attribtue

Do not hesitate if you have more questions

Have a nice day
Olivier.

**Comment by julienanquetil** [comment]
Thanks it's perfect. I'll do it at the office on monday.

**Comment by julienanquetil** [comment]
Hi,
Until now I have no chance to have a functional behat system.... So if you want to merge it "quickly" we have to find someone who can right it / run it

I will try to setup an other VM but need some times (can be week to get it here...)

Thanks

**Comment by solivier** [comment]
Hi @julienanquetil don't worry I will handle the behat for you and run the tests

Thanks again for the PR.

Have a nice day.

File: features/import/product/xlsx/import_products.feature [hunk 1]
```diff
@@ -73,3 +73,18 @@ Feature: Import XLSX products
     Then there should be 1 product
     And the english tablet name of "123" should be "456"
     And the english tablet description of "123" should be "7890"
+
+  Scenario: Successfully import a csv file for a product with a numeric data for an simple select attribute
+    Given the following XLSX file to import:
+      """
+      sku;rating
+      renault-kangoo;5
```
Comment (willy-ahva) [inline features/import/product/xlsx/import_products.feature]: Kangoo FTW!

**Comment by julienanquetil** [comment]
Many thanks
One day I'll have a "dev" VM and not my "uwamp" one...

**Pull request merged**